static/swagger-specs/reactor.yaml

```diff
@@ -11138,6 +11361,533 @@ components:
             type: object
             allOf:
             - $ref: '#/components/schemas/extensionPackagesEntity'
+    extensionPackagesCreateUsageAuthorizationPayload:
+      type: object
+      properties:
+        data:
+          type: object
+          properties:
+            attributes:
+              type: object
+              properties:
+                authorized_org_id:
+                  type: string
+                  description: The authorized Organization ID.
+                  example: "5D1328435BF324E90A49402A@AdobeOrg"
```

Comment:
Generally we put in fake values for this - so I use `{ORG_ID}` here, just like in our ExL API guides.